lib/views/chess_form.dart

```diff
@@ -1,52 +1,45 @@
-import 'package:audioplayers/audioplayers.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import '../bloc/horse_movement_cubit.dart';
+import 'package:audioplayers/audioplayers.dart';
 
-import '../bloc/chess_cubit.dart';
-
-class ChessForm extends StatefulWidget {
-  const ChessForm({super.key});
+import '../bloc/horse_active_cubit.dart';
 
-  @override
-  State<ChessForm> createState() => _ChessFormState();
-}
-
-class _ChessFormState extends State<ChessForm> {
+class ChessForm extends StatelessWidget {
+  ChessForm({super.key});
   final player = AudioPlayer();
 
-  bool active = false;
-
   @override
   Widget build(BuildContext context) {
+    bool active = context.watch<HorseActiveCubit>().state;
     var isPortrait = MediaQuery.of(context).orientation == Orientation.portrait;
-    return BlocBuilder<ChessCubit, List<List<int>>>(
+    return BlocBuilder<HorseMovementCubit, List<List<int>>>(
       builder: (context, state) {
         return Scaffold(
-          backgroundColor: Colors.black,
+          backgroundColor: Colors.blue,
           body: Column(
             children: [
               const Spacer(),
               ...List.generate(
-                  //TODO: remove this magic number
```

Comment:
This is not magic number because chess has a fixed number 8 rows and 8 columns 
**Go learn to play Chess broo**

Reply:
the you should to write it as 
const row = 8;
const columns = 8 ; 
or even write comment  to clarify that